lib/foreman_ansible/register.rb

```diff
@@ -3,6 +3,82 @@
 Foreman::Plugin.register :foreman_ansible do
   requires_foreman '>= 2.6'
 
+  settings do
+    category :ansible, N_('Ansible') do
+      setting 'ansible_ssh_private_key_file',
+              type: :string,
+              description: N_('Use this to supply a path to an SSH Private Key '\
+                               'that Ansible will use in lieu of a password '\
+                               'Override with "ansible_ssh_private_key_file" '\
+                               'host parameter'),
+              default: '',
+              full_name: N_('Private Key Path')
+      setting 'ansible_connection',
+              type: :string,
+              description: N_('Use this connection type by default when running '\
+                               'Ansible playbooks. You can override this on hosts by '\
+                               'adding a parameter "ansible_connection"'),
+              default: 'ssh',
+              full_name: N_('Connection type')
+      setting 'ansible_winrm_server_cert_validation',
+              type: :string,
+              description: N_('Enable/disable WinRM server certificate '\
+                               'validation when running Ansible playbooks. You can override '\
+                               'this on hosts by adding a parameter '\
+                               '"ansible_winrm_server_cert_validation"'),
+              default: 'validate',
+              full_name: N_('WinRM cert Validation')
+
+      setting 'ansible_verbosity',
+              type: :integer,
+              description: N_('Foreman will add this level of verbosity for '\
+                               'additional debugging output when running Ansible playbooks.'),
+              default: '0',
+              full_name: N_('Default verbosity level'),
+              value: nil,
+              collection:
+        { '0' => N_('Disabled'),
+          '1' => N_('Level 1 (-v)'),
+          '2' => N_('Level 2 (-vv)'),
+          '3' => N_('Level 3 (-vvv)'),
+          '4' => N_('Level 4 (-vvvv)') }
```

Comment:
This still needs to be a proc (or lambda) even if the result is static Hash, the renderer expects it to be always a `proc`
```suggestion
lambda do
        { '0' => N_('Disabled'),
          '1' => N_('Level 1 (-v)'),
          '2' => N_('Level 2 (-vv)'),
          '3' => N_('Level 3 (-vvv)'),
          '4' => N_('Level 4 (-vvvv)') }
 end
```